spec/controllers/users/webauthn_setup_controller_spec.rb

```diff
@@ -364,16 +364,6 @@
       end
     end
 
-    context 'Multiple MFA options turned off' do
```

Comment:
Removed since:

- `context` is for a scenario which no longer exists (a feature flag that's been removed)
- The written test description is the opposite of what it asserts (describes being sent for second MFA, asserts being sent to account page)
- There's already a test case above that asserts the expected behavior for a single selection in the multi-MFA setup flow ([source](https://github.com/18F/identity-idp/blob/ad1e9f03bbef7f7ddf2d1bef7e7d7019ceb99ba7/spec/controllers/users/webauthn_setup_controller_spec.rb#L263-L271))